react/README.md

```diff
@@ -5,7 +5,7 @@
 ## Table of Contents
 
   1. [Basic Rules](#basic-rules)
-  1. [Class vs `React.createClass`](#class-vs-reactcreateclass)
+  1. [Class vs `React.createClass` vs stateless](#class-vs-reactcreateclass-vs-stateless)
```

Comment:
it's unfortunate that this will break the link - however since the title on line 28 didn't change, are you sure this link works now?